lib/ruby_llm/active_record/acts_as.rb

```diff
@@ -114,6 +114,17 @@ def with_temperature(temperature)
         self
       end
 
+      # Specifies the response format for the chat (JSON mode or JSON schema)
+      # @param response_format [Hash, String, Symbol] The response format, either:
+      #   - :json for simple JSON mode
+      #   - JSON schema as a Hash or JSON string for schema-based output
+      # @param strict [Boolean] Whether to enforce model compatibility (default: true)
+      # @return [self] Chainable chat instance
+      def with_response_format(response_format, strict: true)
```

Comment:
Can we add `alias with_response_schema with_response_format`?

Reply:
I just noticed the comments above that it was originally `with_response_schema`. Going to comment above there because I think your original instinct was much better!

Reply:
I like `with_response_format`, let's stick to that.

Reply:
Also, thjis might be a schema or a format. JSON mode does not have a schema.